CHANGES.md

```diff
@@ -10,7 +10,8 @@ Change Log
 * Added `maximumHeight` option to `Viewer.flyTo`. [#2868](https://github.com/AnalyticalGraphicsInc/cesium/issues/2868)
 * Added picking support to `UrlTemplateImageryProvider`.
 * Added ArcGIS token-based authentication support to `ArcGisMapServerImageryProvider`.
-* `ArcGisMapServerImageryprovider` now supports a proxy for `pickFeatures` requests.
+* `ArcGisMapServerImageryProvider` now supports a proxy for `pickFeatures` requests.
+* Added `MapboxImageryProvider`
```

Comment:
Not everyone knows what Mapbox is, so I would expand this to something like "Added `MapboxImageryProvider` to make it easier to load imagery from [Mapbox](http://mapbox.com)"